tuxemon/item/conditions/has_status.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from tuxemon.item.itemcondition import ItemCondition
+from tuxemon.monster import Monster
+
+
+@dataclass
+class HasStatusCondition(ItemCondition):
+    """
+    Checks if the creature has a status or not.
+
+    Parameters:
+    expected: true or false
+
+    Example:
+    "conditions": [
+        "has_status target,true"
```

Comment:
We can already do not has_status, so we don't need the second param

Reply:
True